src/interfaces/agm-ext-eval-interface.ts

```diff
@@ -14,10 +15,11 @@ export interface AgmExtEvalIF {
 
 /** An empty AGM Extension Evaluation object. Note: don't assign this - make a copy instead. */
 export const EmptyAgmExtEval: AgmExtEvalIF = {
+  isEligible: null,
   isGoodStanding: null,
   incorporationDate: null,
   isFirstAgm: null,
   agmYear: null,
-  extensionDuration: NaN,
```

Comment:
Both NaN and null as valid "non-numeric" values for a number, so your change is fine.

PS You can check is a variable is a number with `isNaN(x)`.

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/607db820-a879-4712-99cb-0ba583234209)